Assets/MRTK/Providers/Oculus/XRSDK/MRTK.ThirdParty/MRTK-Quest-Utils/Scripts/HandInput/HandPoseUtils.cs

```diff
@@ -0,0 +1,198 @@
+﻿//------------------------------------------------------------------------------ -
+//MRTK - Quest
+//https ://github.com/provencher/MRTK-Quest
+//------------------------------------------------------------------------------ -
+//
+//MIT License
+//
+//Copyright(c) 2020 Eric Provencher
+//
+//Permission is hereby granted, free of charge, to any person obtaining a copy
+//of this software and associated documentation files(the "Software"), to deal
+//in the Software without restriction, including without limitation the rights
+//to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//copies of the Software, and to permit persons to whom the Software is
+//furnished to do so, subject to the following conditions :
+//
+//The above copyright notice and this permission notice shall be included in all
+//copies or substantial portions of the Software.
+//
+//THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+//AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//SOFTWARE.
+//------------------------------------------------------------------------------ -
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    public static class HandPoseUtils
```

Comment:
It might be worth commenting both the class here and some of the methods to mention how they differ from other gesture/pose stuff in the rest of the MRTK - i.e. it might be confusing now that there's grab gestures/events and also this grab helper function, which are completely unrelated.

Documenting the purpose of when this can be useful (i.e. systems that don't have similar native gesture support or whatever) can help avoid complications in the future where someone is like "hey I'm getting a grab event here but then I ask this HandPoseUtils and it says it's not grabbing!"